packages/di/README.md

```diff
@@ -216,3 +216,27 @@ If at some point you want to create registry that wan't be overrided just call t
 ```ts
 const boldRegistry = new Registry({ id: cnApp(), overridable: false })
 ```
+
+## Extends components
+
+Components inside registry can be extends (e.g. for experiments) by wrapping `withRegistry(...)(App)` with another registry.
```

Comment:
You can extend (e.g. for experiments) a component using `withBase(...)`-form in override regjstry: